include/start-container.sh

```diff
@@ -34,8 +34,17 @@ if [ -n "$1" ]; then
         services)
             /opt/vespa/bin/vespa-start-services
             ;;
+        services,configserver | configserver,services)
```

Comment:
In this mode it will be required to set the `VESPA_CONFIGSERVERS`. Is this as intended ?

Reply:
Good question! For the usecase I need this (multinode guides) it will be required. The question is if we should allow a blank `VESPA_CONFIGSERVERS` for the singlenode install. We can argue that it is OK to require a value for this new setting, for now, we can always lift this later